### DISCUSSION TITLE
WinModder/PowerShell Scripts Suggestions Discussion

### DISCUSSION THREAD

**Discussion by noobtodamax** [builtbybel/Bloatynosy#70] [Ideas]
ThisisWin11 user here,
I appreciate the WinModder idea for really small powershell tweaks/scripts. I have a suggestion for one, but I couldn't find the place to offer these types of suggestions so I figured I'd make one... :)

**Comment by noobtodamax** [comment]
One thing I've been looking for: a script that, when creating shortcuts for desktop, the system will no longer add " - shortcut" to the file name.
![ArchiveShortcutPic](https://github.com/builtbybel/BloatyNosy/assets/41933987/268f0482-1d9d-4d1a-b3e2-92ac4fd921e4)

**Reply by jimbrig** [reply]
@noobtodamax this can be achieved via the registry:

- Key   : `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\NamingTemplates`
- Name  : `ShortcutNameTemplate` 
- Value : `"%s.lnk"`
- Type  : `REG_SZ`

resulting in this:

![image](https://github.com/user-attachments/assets/a418093a-735b-4b38-8b02-4fbc4cf907d5)

another option is to simply utilize a registration (`.reg`) script:

```registry
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\NamingTemplates]
"ShortcutNameTemplate"="%s.lnk"
```

Download: [ExplorerShortcutNamingTemplate.reg](https://github.com/user-attachments/files/16368657/ExplorerShortcutNamingTemplate.reg.txt)

> [!WARNING]
> Due to GitHub File Upload restrictions you will first need to rename `ExplorerShortcutNamingTemplate.reg.txt` to `ExplorerShortcutNamingTemplate.reg`.

> [!NOTE]
> You can alter the template as you want, for example you could make it `"Shortcut to %s.lnk"` to display as `Shortcut to MyApp` in explorer (assuming the file is named `MyApp`).

To restore default behavior simply delete the registry key or unset its value.

***

P.S. You can also remove the shortcut arrow icon from the thumbnail if so desired.
